receiver/aerospikereceiver/cluster/cluster_test.go

```diff
@@ -27,5 +27,5 @@ func TestCluster_GetNodes(t *testing.T) {
 	}
 
 	actualNodes := testCluster.GetNodes()
-	require.Equal(t, len(actualNodes), len(nodes))
+	require.Len(t, nodes, len(actualNodes))
```

Comment:
i understand this is what testify wants, but i personally find it harder to read than it was before